### DISCUSSION TITLE
训练文本识别模型时占用GPU的Memory-Usage，但是利用率一直只有百分之几。训练文本检测模型时是正常的。

### DISCUSSION THREAD

**Discussion by juneana123** [PaddlePaddle/PaddleOCR#12504] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by hezhenname** [comment]
应该不是一直吧，我训练的时候随机查过，有时候会是百分之几，但是有时候还是会跑到98%或者100%，可能是处理图片的时候才调GPU

**Comment by jingmingtao** [comment]
> 应该不是一直吧，我训练的时候随机查过，有时候会是百分之几，但是有时候还是会跑到98%或者100%，可能是处理图片的时候才调GPU

大佬我请问你一个问题吧，就是我想问的是图片标注，就是用paddlelbl，我的指标hemean一直为0，是不是图片小，字大，行少导致的阿
